packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -194,10 +194,18 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
       className,
       minDate,
       maxDate,
-      calendarProps
+      calendarProps,
+      calloutProps
     } = this.props;
     const { isDatePickerShown, formattedDate, selectedDate, errorMessage } = this.state;
 
+    let calendarStrings = null;
+    if (calendarProps && calendarProps.strings) {
+      calendarStrings = calendarProps.strings;
+    } else if (DEFAULT_STRINGS) {
+      calendarStrings = DEFAULT_STRINGS;
+    }
```

Comment:
The old code seems cleaner, why do you need to change?

Reply:
If someone uses calendarProps.strings then Default_strings overrides it.  I suppose this is ok, since Default strings is practically the same thing.